src/tests/ftest/dfuse/sparse_file.py

```diff
@@ -77,15 +77,15 @@ def test_sparsefile(self):
         fsize_after_truncate = get_remote_file_size(self.hostlist_clients[0], sparse_file)
         self.log.info("File size (in bytes) after truncate: %s", fsize_after_truncate)
         # verifying the file size got set to desired value
-        self.assertTrue(fsize_after_truncate == self.space_before)
+        self.assertEqual(fsize_after_truncate, self.space_before)
 
         # write to the first byte of the file with char 'A'
         dd_first_byte = "echo 'A' | dd conv=notrunc of={} bs=1 count=1".format(sparse_file)
         self.execute_cmd(dd_first_byte)
         fsize_write_1stbyte = get_remote_file_size(self.hostlist_clients[0], sparse_file)
         self.log.info("File size (in bytes) after writing first byte: %s", fsize_write_1stbyte)
         # verify file did not got overwritten after dd write.
-        self.assertTrue(fsize_write_1stbyte == self.space_before)
+        self.assertEqual(fsize_write_1stbyte, self.space_before)
```

Comment:
Could we add a descriptive message here that will be displayed if the assert fails?